### PR TITLE
[OP-1121] Correct the jar version into dockerfile.backend

### PR DESCRIPTION
**Pull request by Tudor44**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -90,10 +90,10 @@ You can see Swagger Api Documentation at: http://localhost:8080/swagger-ui/
 
 Make sure you have docker with docker-compose installed, then run the following commands:
 
-- copy `dotenv` file into `.env` and set variables as needed (the SHA-256 jwt token is needed)
+- rename the `dotenv` file in `.env` and set variables as needed (the SHA-256 jwt token is needed)
```
Comment (mwithi) [inline README.md:93]: renaming would result in a different git status and may be commited by mistake

Comment (Tudor44) [reply README.md:93]: > renaming would result in a different git status and may be commited by mistake

Yes, I would say rename .dotenv, but keeping the original .dotenv versioned as is into current branch. 
It should be useful a .gitignore entry for .env

Comment (mwithi) [reply README.md:93]: oh, I supposed was already there in .gitignore like did in other repos, where we also use the same instructions: dotenv and copy to .env

could you please revert this line in README and add .env to .gitignore?

Comment (Tudor44) [reply README.md:93]: > oh, I supposed was already there in .gitignore like did in other repos, where we also use the same instructions: dotenv and copy to .env
> 
> could you please revert this line in README and add .env to .gitignore?

Ok sure, I ll add it in develop only?

**Comment by mwithi** [comment]
was this PR meant to be closed?

File: README.md (hunk 1 above)
Comment (Tudor44) [reply README.md:93]: Added new PR with JIRA assocations.

**Comment by mwithi** [comment]
> was this PR meant to be closed?

ok, sorry, I've seen the new PR now

**Comment by Tudor44** [comment]
> was this PR meant to be closed?

Hi @mwithi , sorry I added it in a new PR

**Pull request closed without merging**